piqueserver/player.py

```diff
@@ -333,12 +335,25 @@ def ban(self, reason=None, duration=None):
                 self.protocol.add_ban(self.address[0], reason, duration,
                                       self.name)
 
-    def send_lines(self, lines: List[str]) -> None:
+    def send_lines(self, lines: List[str], type: str = 'unknown') -> None:
```

Comment:
I would rename type to key, as type is ambiguous and collides with the built-in type function